main/REPL/timerServices.md

```diff
@@ -0,0 +1,75 @@
+# Timer Service Objects
+
+`home` has two timer service objects: 
+- `localTimerService` is from the local vat.
+- `chainTimerService` is from the chain (if present). 
+
+In general, you should use the chain timer for contracts. But it's more efficient to 
+use the local timer for objects that don't need consensus or consistency. **tyg todo: Examples of such local prefered objects?**
+
+Both timer services let you:
+- Get the start time of the current block. **tyg todo: Is this the case for local?**
+- Schedule `wake()` calls.
+- Remove scheduled `wake()` calls.
+- Create a repeater for scheduling of events at regular intervals. 
+
+With respect to using them, the only difference between the two timer
+services is whether you preface their identical command set with
+`localTimerService` or `chainTimerService` 
+
+## `E(home.<chain or local>TimerService).getCurrentTimestamp()`
+- Returns: If `chainTimerService`, the current block's start time **tyg todo: This
+appears  to be the number of milliseconds since the
+current block was created?** 
+- Returns: If `localTimerService(), **tyg todo:
+unclear what it's the start time of?** start time in milliseconds.
+```js
+command[1] E(home.chainTimerService).getCurrentTimestamp()
+history[1] 1608523721
+command[2] E(home.localTimerService).getCurrentTimestamp()
+history[2] 1340435997
+```
+
+## `E(home.<chain or local>TimerService.(setWakeup(baseTime, handler)`
+- `baseTime` `{ integer }` 
+- `handler` `{ Handler }`
+- Returns: `{ Integer }` 
+
+Calls the specified handler the specified number of seconds after executing this call.
+
+Returns the time at which the call is scheduled to happen. **tyg todo: In what format?**
+
+```js
+const handler = harden({wake(now) { console.log(`woke up ${now}`); }});
```

Comment:
Apparently `const` is broken now. And you're halfway between `function` and `lambda` syntax. I suggest `lambda`.

```suggestion
handler = harden({ wake: now => { console.log(`woke up ${now}`); }})
```